models/profile.js

```diff
@@ -0,0 +1,19 @@
+var mongoose = require('mongoose'),
+  Schema = mongoose.Schema;
+
+var Projects=require("./projects.js");
+var Vacation=require("./vacation.js");
+
+var ProfileSchema = new Schema({
+    name: String,
+    githubUserName:String,
+    gitHubLink:String,
+    currentcity:String,
+    hobbies:String,
+    projects:[ Projects.schema ],
```

Comment:
extra spaces in between brackets?

Reply:
that's just a indentaion. not a big deal